### PR TITLE
Changed text regarding modification of manifest files. BZ1904138

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
…BZ1904138.

https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-network-customizations.html#modifying-nwoperator-config-startup_installing-aws-network-customizations (also in 4.4, 4.5 and 4.6)

Text change in modules/nw-modifying-operator-install-config.adoc in the "IMPORTANT" box:

"Modifying the OpenShift Container Platform manifest files directly is not supported."

Changed to:
"Modify the OpenShift Container Platform manifest files only after installation completes."

### CODE REVIEW

**Comment by GroceryBoyJr** [comment]
Link for the preview of the change:
https://deploy-preview-31084--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-network-customizations.html#modifying-nwoperator-config-startup_installing-aws-network-customizations

**Comment by GroceryBoyJr** [comment]
Yunfei - I agree with you. Also, discussed with support and got the following recommendation. I believe it will work better for the text in the information box because it makes clear the original question of what we can SUPPORT. Please look at this and tell me:


_Modifying the OpenShift Container Platform manifest files created by the installation program is not supported. Applying a manifest file that you create, as in the following procedure, is supported._


Yunfei, I would very much like to install that text into the IMPORTANT text block, please. THANK YOU for your help on this, much appreciated!!!

James Brigman

**Comment by GroceryBoyJr** [comment]
Submitting for merge. 
BZ1904138: https://bugzilla.redhat.com/show_bug.cgi?id=1904138
Preview: https://deploy-preview-31084--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-network-customizations.html#modifying-nwoperator-config-startup_installing-aws-network-customizations
(look for the "salmon colored" INFORMATION box right under "Modifying Advanced Network Configuration Parameters")
QE approval has been received.

File: modules/nw-modifying-operator-install-config.adoc [hunk 1]
```diff
@@ -28,7 +28,9 @@ parameter.
 
 [IMPORTANT]
 ====
-Modifying the {product-title} manifest files directly is not supported.
+Modifying the {product-title} manifest files created by the installation program
+is not supported. Applying a manifest file that you create, as in the following
```
Comment (bergerhoffer) [inline modules/nw-modifying-operator-install-config.adoc:32]: The text looks good to me, but the lines should not be hard wrapped. See here in our guidelines: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines

**Review by bergerhoffer — COMMENTED** [review]
Once you upwrap the line, this lgtm!

File: modules/nw-modifying-operator-install-config.adoc (hunk 1 above)
Comment (GroceryBoyJr) [reply modules/nw-modifying-operator-install-config.adoc:32]: Thank you Andrea!

**Comment by bergerhoffer** [comment]
Thanks for the update to fix the line wrapping. LGTM now, merging since QE approval has been received.

**Pull request merged**